resources/views/media/manager.blade.php

```diff
@@ -293,7 +293,8 @@
 
                 <div class="modal-body">
                     <h4>{{ __('voyager::media.destination_folder') }}</h4>
-                    <select class="form-control">
+                    <select class="form-control" v-model="modals.move_files.destination">
+                        <option value="" disabled>Select Destination</option>
```

Comment:
This hardcoded string needs to be translated.
For simplicity I suggest 
```suggestion
                        <option value="" disabled>{{ __('voyager::media.destination_folder') }}</option>
```